Sprint-1/implement/max.js

```diff
@@ -1,4 +1,7 @@
+//const ITEMS = [10,50,60,"a"];
 function findMax(elements) {
+    let numericItems = elements.filter(value => typeof value === 'number');
+    return numericItems.length > 0 ? Math.max(...numericItems) : -Infinity;
 }
-
+//console.log(findMax(ITEMS));
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
findMax([-Infinity])
findMax([-Infinity, 0])
findMax([NaN])
findMax([0, NaN, 1])
```